docs/quick_start.rst

```diff
@@ -145,7 +145,7 @@ Now that there are some packages in the workspace, Catkin has something to build
 .. note::
 
     Catkin utilizes an "out-of-source" and "aggregated" build pattern. This
-    means that not only will temporary or final build products never be placed
+    means that not only will temporary or final build products ever be placed
```

Comment:
Is this right? Read it aloud I would have said never, but then again it could be more grammatically correct to say ever. Perhaps it should be reworded entirely to avoid the risk of a double negative.